packages/client/src/main.ts

```diff
@@ -31,21 +36,53 @@ const injectHeadAssets = () => {
     });
 };
 
-const init = () => {
-    initParams();
-    injectHeadAssets();
-    initHistoryEvents();
-    initScrollToEvents();
-
+const startTrackingServices = () => {
+    console.log("Starting tracking services");
```

Comment:
Skal vi ha console.logging fortsatt? I testperioden kanskje?

Reply:
Ja, jeg tenkte å ha de med for å understreke at "nå stopper vi å spore" for de 4,5 personene som faktisk går inn og kontrollerer. Og litt for innkjøring også. Men er åpen for innspill!